src/item.cpp

```diff
@@ -2361,9 +2365,15 @@ std::string item::display_name( unsigned int quantity ) const
     bool has_ammo = is_ammo_container() && !contents.empty();
     bool contains = has_item || has_ammo;
     bool show_amt = false;
+    bool is_money = ( ammo_type() == "money");
 
     // We should handle infinite charges properly in all cases.
-    if( contains ) {
+    if ( is_money ) {
+        for (auto &elem : contents) {
```

Comment:
Are you sure `is_money` is evaluating to `true` for stacked cash cards?  If it fails and the `else if (contains)` check below runs, it would have the behavior you are describing.

Reply:
Yeah. 
Originally, cash cards were handled by the "} else if( ammo_capacity() > 0 ) {" line, as cash cards have "type": "TOOL", the if( contains ) line is false for cash cards.